scripts/version.ts

```diff
@@ -0,0 +1,27 @@
+import "reflect-metadata";
+import * as typeGraphQL from "type-graphql";
```

Comment:
This introduces a catch-22: this script is run before its dependency `type-graphql` executes.

I suggest either split the versions across the two version files or store the version info in a common non-code file such as JSON.